pkgs/by-name/vk/vkdevicechooser/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  writeText,
+  meson,
+  vulkan-headers,
+  vulkan-utility-libraries,
+  ninja,
+  jq,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "vkdevicechooser";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "jiriks74";
+    repo = "vkdevicechooser";
+    rev = version;
+    hash = "sha256-j4hefarOQ3q0sKkB2g/dO2/4bSYzt4oxmCna0nMAjFk=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    jq
+  ];
+
+  buildInputs = [
+    vulkan-headers
+    vulkan-utility-libraries
+  ];
+
+  # Help vulkan-loader find the validation layers
```

Comment:
"the validation layers" is incorrect here.

Reply:
I changed that to "the layer".